components/OpenIbox.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
```

Comment:
There's a typo in the filename. As defined in the issue, let's call this `OpenEmailProviderButton`

Reply:
Done